docs/spec/index.html

```diff
@@ -1,740 +1,952 @@
+      presentations relate to the same
+      <em>type</em>
+      of workflow.
+    </p>
+    <p>
+      Workflow <em>instances</em> indicate that separate individual
+      presentations are part of the same
+      <em>instance</em> of a specified workflow <em>type</em>.
+    </p>
 
+    <section class="informative">
+      <h3>Workflows for US Customs and Border Protection (US CBP)</h3>
+      <img src="resources/workflow-def-ex.png" alt="workflow-def-ex"
+        width="75%" />
       <p>
-        To contribue to this vocabulary or reference technical details related
-        to the project, please reference the primary README located on
-        <a href="https://github.com/w3c-ccg/traceability-interop/">github</a>
+        In the above example, two separate 7501 Entry Summaries are
+        presented to US CBP at different points in time, and related to
+        different imports. By specifying the same workflow
+        <em>definition</em>, it is made clear to US CBP that both presentations
+        relate to the same <em>type</em> of
+        import workflow.
       </p>
-
+      <img src="resources/workflow-inst-ex.png" alt="workflow-inst-ex"
+        width="75%" />
       <p>
-        Please
-        <a href="https://github.com/w3c-ccg/traceability-interop/issues/"
-        >open an issue</a
-        >, if you wish to collaborate on this specification.
+        The majority of data elements in a "10+2" Importer Security
+        Filing must be presented to CBP no later than 24 hours before
+        vessel loading, and the remaining data elements no later than 24
+        hours before arrival. Specifying the same workflow
+        <em>instance</em> indicates to US CBP that the two presentations are
+        part of the same import workflow.
       </p>
-
       <p>
-        You may also reach out via the mailing list:
-        <a href="mailto:public-credentials@w3.org">public-credentials@w3.org</a>
-        (<a href="mailto:public-credentials-request@w3.org?subject=subscribe"
-        >subscribe</a
-        >,
-        <a href="https://lists.w3.org/Archives/Public/public-credentials/"
-        >archives</a
-        >)
+        A workflow definition indended for use with US CBP imports
+        <em>MUST</em>
+        have a corresponding plain text
+        <em>workflow manifest document</em> that may be uniquely
+        identified, referenced, and retrieved by a third party in a
+        publicly accessible and documented manner by the UUID of the
+        workflow definition.
       </p>
-    </section>
-
-    <section class="informative">
-      <h2>Introduction</h2>
-
       <p>
-        This specification describes how verifiable data can be shared securely
-        over an authenticated channel. Specifically, a set of OpenAPI HTTP
-        endpoints is defined, allowing authenticated data senders and receivers
-        to exchange Verifiable Presentations.
+        This workflow manifest document <em>MUST</em> list each required
```

Comment:
I like these changes, but prefer for an issue to be linked, and for them not to be sprinkled in such a large PR.

Reply:
Created issue #495 to track workflow manifest document definition

Reply:
Please add a link to the spec, I filed this because I am not comfortable with the changes without a visible spec change tracking that they don't have consensus yet.

Reply:
@OR13 I've added a link to #495 above.